ale/base/type_sensor.py

```diff
@@ -49,6 +49,21 @@ def ephemeris_time(self):
         """
         return np.linspace(self.ephemeris_start_time,  self.ephemeris_stop_time, self.image_lines / 64)
 
+    @property
+    def center_ephemeris_time(self):
+        """
+        Returns the sum of the starting ephemeris time and half of the line exposure
+        duration. Expects ephemeris start time and line exposure duration to be
+        defined. These should be double precision numbers containing the ephemeris start and
+        line exposure duration of the image.
+
+        Returns
+        -------
+        : double
+          Center ephemeris time for an image
+        """
+        return self.ephemeris_start_time + self.line_exposure_duration / 2
```

Comment:
For line scan sensors, the end time should be `ephemeris_start_time + lines * line_exposure_duration`